pkg/repository/repository.go

```diff
@@ -12,7 +12,9 @@ import (
 	"github.com/docker/distribution/registry/api/errcode"
 	v2 "github.com/docker/distribution/registry/api/v2"
 	"github.com/google/go-containerregistry/pkg/crane"
+	crname "github.com/google/go-containerregistry/pkg/name"
```

Comment:
`crname` required?

Reply:
Yes, there is some conflict with `name` if we don't use `crname`.